tests/test_model.py

```
@@ -385,73 +385,73 @@ def test_bias_scale_none_matches_default(
-        """``bias_scale`` applies verbatim to bias sigma even when ``use_layerwise_scaling`` shrinks weight sigma."""
+        """``bias_std`` applies verbatim to bias sigma even when ``use_layerwise_scaling`` shrinks weight sigma."""
         bnn = BayesianNeuralNetwork.cold_start(
             n_features=n_features,
             hidden_dim_list=hidden_dim_list,
             dist_type=dist_type,
             dist_params_init=self._dist_params(dist_type, sigma, nu),
             use_layerwise_scaling=True,
-            bias_scale=bias_scale,
+            bias_std=bias_std,
         )
         fan_ins = [n_features] + hidden_dim_list
```

Ruff (0.15.13)

[warning] Consider `[n_features, *hidden_dim_list]` instead of concatenation

Replace with `[n_features, *hidden_dim_list]`

(RUF005)